contents/tutorials/fake-door-test.md

```diff
@@ -0,0 +1,139 @@
+---
+title: How to run a fake door test
+date: 2023-08-09
+author: ["ian-vanagas"]
+showTitle: true
+sidebar: Docs
+featuredImage: ../images/tutorials/banners/tutorial-15.png
+tags: ['surveys', 'actions']
+---
+
+A fake door test is creating a real UI for a non-existent product or feature to gauge interested in it. Doing one can help you validate what you plan on building. In this tutorial, we build an app, set up a fake door test, then implement surveys to get more feedback.
+
+## Creating an app and adding PostHog
+
+We are going to build a basic app to run our fake door test on. We do this with Next.js, which requires installing Node. After doing so, run the command below, select **No** for TypeScript, **Yes** for `use app router`, and the defaults for every other option.
+
+```bash
+npx create-next-app@latest fake-door
+```
+
+After doing this, go into your newly created `fake-door` folder and install `posthog-js`.
+
+```bash
+cd fake-door
+npm i posthog-js
+```
+
+After installing `posthog-js`, set up the PostHog initialization. This requires going to our `app` folder, creating a `providers.js` file, adding our PostHog initialization behind a window check with `opt_in_site_apps: true` (for later), and creating a provider component like this:
+
+```js-web
+// app/providers.js
+'use client'
+import posthog from 'posthog-js'
+import { PostHogProvider } from 'posthog-js/react'
+
+if (typeof window !== 'undefined') {
+  posthog.init('<ph_project_api_key>', {
+    api_host: '<ph_instance_address>',
+    opt_in_site_apps: true
+  })
+}
+
+export function PHProvider({ children }) {
+  return <PostHogProvider client={posthog}>{children}</PostHogProvider>
+}
+```
+
+With our `PHProvider` component created, we can import it and wrap our main app in it in `layout.js`.
+
+```js-web
+// app/layout.js
+import './globals.css'
+import { PHProvider } from './providers'
+
+export default function RootLayout({ children }) {
+  return (
+    <html lang="en">
+      <PHProvider>
+        <body>{children}</body>
+      </PHProvider>
+    </html>
+  )
+}
+```
+
+With this done, PostHog starts autocapturing events and our app is ready for a fake door test.
+
+## Creating our fake door test
+
+Creating a fake door test is intentionally simple. There are many ways to do it, but we start with a basic one. Add a button to our app that says "Try our new feature."
```

Comment:
In our example, we'll create a "fake" sign in button. When clicked, instead of signing you in, we'll show a message that reads "Thanks for your interest! This feature is coming soon".